### PR TITLE
Port redirects from current website

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: .netlifyredirects [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/internal   /api
+/tomster    /mascots
+/tomster/*  /mascots/:splat
+/dasboard   /statusboard
+/guides     https://guides.emberjs.com/
+/guides/    https://guides.emberjs.com/
```
Comment (mansona) [inline .netlifyredirects]: This needs to have "splatributes" added to the definition. You can see the documentation here: https://www.netlify.com/docs/redirects/#splats

File: .netlifyredirects [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/internal   /api
+/tomster    /mascots
+/tomster/*  /mascots/:splat
+/dasboard   /statusboard
+/guides     https://guides.emberjs.com/
```
Comment (mansona) [inline .netlifyredirects]: you don't need both trailing and non trailing slashes in the definition: https://www.netlify.com/docs/redirects/#trailing-slash

Comment (locks) [reply .netlifyredirects]: We do need splat and non-splat though, confirm?

File: .netlifyredirects [hunk 3]
```diff
@@ -0,0 +1,4 @@
+/internal   /api
+/tomster/*  /mascots/:splat
+/dasboard   /statusboard
+/guides/*   https://guides.emberjs.com/:splat
```
Comment (mansona) [inline .netlifyredirects]:
```suggestion
/guides/*   https://guides.emberjs.com/release/:splat
```

Looking at one of the examples of a "bad link" that this is supposed to fix https://emberjs.com/guides/models/ it seems like we need to include `release` in the redirect 🤔 

without this improvement https://deploy-preview-133--ember-beta-website.netlify.com/guides/models/ redirects to https://guides.emberjs.com/models/ which is a 404

**Pull request merged**